apps/meteor/client/views/setupWizard/hooks/useStepRouting.ts

```diff
@@ -22,12 +23,12 @@ export const useStepRouting = (): [number, Dispatch<SetStateAction<number>>] =>
 	});
 
 	useEffect(() => {
-		if (hasAdminRole && currentStep === 1) {
-			setCurrentStep(2);
+		if ((currentStep === 1 || currentStep === 2) && hasOrganizationData) {
+			setCurrentStep(3);
 		}
 
-		setupWizardRoute.replace({ step: String(currentStep) });
-	}, [setupWizardRoute, currentStep, hasAdminRole]);
+		router.navigate(`/setup-wizard/${String(currentStep)}`);
+	}, [router, currentStep, hasAdminRole, hasOrganizationData]);
```

Comment:
`hasAdminRole` is it really necessary?

Reply:
in fact, on refactoring added hasAdminRole on checks!